test/evergreen.yml

```diff
@@ -44,7 +44,38 @@ functions:
     command: shell.exec
     params:
       script: |
+        set -o errexit
+        set -o verbose
         git clone https://github.com/wiredtiger/mongo-tests
+  "fetch mongodb repo" :
+    command: shell.exec
+    params:
+      script: |
+        set -o errexit
+        set -o verbose
+        git clone https://github.com/mongodb/mongo
+  "merge mongodb wt" :
+    command: shell.exec
+    params:
+      script: |
+        set -o errexit
+        set -o verbose
+        cp -a wiredtiger mongo/src/third_party/
+  "compile mongodb" :
+    command: shell.exec
+    params:
+      shell: bash
+      working_dir: "mongo"
+      script: |
+        set -o errexit
+        set -o verbose
+        virtualenv -p python3 venv
+        source venv/bin/activate
+        pip3 install requirements_parser
+        pip3 install -r etc/pip/compile-requirements.txt
+        sudo apt install -y ccache
+        ./buildscripts/scons.py --variables-files=etc/scons/mongodbtoolchain_stable_gcc.vars --link-model=dynamic --ninja generate-ninja ICECC=icecc CCACHE=ccache
+        ninja -j$(nproc --all) install-mongod
```

Comment:
I guess the contents are inherited from mongo server project. It should work for now, but I'm a bit concerned about the potential future maintenance burden (when mongo server updates their building commands/options etc).

Reply:
I followed instructions given by the server team. We definitely need to keep this test in mind in case the building instructions would change.